CHANGELOG.rst

```diff
@@ -2,19 +2,35 @@
 Changelog
 =========
 
+`2023.7 <https://github.com/python/python-docs-theme/releases/tag/v2023.7>`_
+----------------------------------------------------------------------------
+
+- Do not append asset hashes on Sphinx 7.1+ (#137)
+   Contributed by Pradyun Gedam
+- Use correct block, enable built-in slash keypress (#141)
+   Contributed by Mike Fiedler
+- Sphinx 6.2 fix: add ``nav.contents`` where that ``div.topic`` is used (#138)
+   Contributed by Hugo van Kemenade
+- Dark mode: fix contrast for C++ specific styling (#133)
+  Contributed by Hugo van Kemenade
+- Don't let long code literals extend beyond the right side of the screen (#139)
+   Contributed by Hugo van Kemenade
+- Test 3.12 branch, now `main` is 3.13 (#140)
```

Comment:
More descriptive ('branch' implicitly refers to CPython here, but it isn't very clear)
```suggestion
- Test with Python 3.12 (#140)
```